docs/group_guidelines/git_standards.md

```diff
@@ -0,0 +1,86 @@
+# Git - Standard Practices
+The purpose of this document is to provide our team with a set of standards to follow throughout our capstone project.
+With this document, we hope to ensure that we stay organized and consistent throughout the term.
+
+# Branches
+
+We will be following a technique called feature branches, where we create a new branch for every feature which is to be
+developed. This allows us to prevent merge conflicts, and closely monitor each new feature we need to create. As well,
+feature branches ensure we are dividing properly dividing issues into achievable tasks. Once the feature has been 
```

Comment:
Please clarify this sentence. I think the word "dividing" was accidentally added twice

Reply:
You are very right